kitsune/sumo/utils.py

```diff
@@ -328,3 +329,16 @@ def is_ratelimited(request, name, rate, method=['POST'], skip_if=lambda r: False
             Record.objects.info('sumo.ratelimit', '{key} hit the rate limit for {name}',
                                 key=key, name=name)
     return request.limited
+
+
+def get_browser(user_agent):
+    """Get Browser Name from User Agent"""
+
+    user_agent = str(user_agent)
```

Comment:
Yap. The user agent is always a string but I got a error while testing in my local. Maybe its otherwise. 
Also I thought that anyone may pass other things through the user agent. But didn't have much thinking. I googled and it seems that only string can be passed by header. Also corrected in next commit!